### DISCUSSION TITLE
DISCUSSION: Seattle DataFusion Meetup (April 23, 2026)

### DISCUSSION THREAD

**Discussion by tbar4** [apache/datafusion#13500] [General]
Is there any interest for a Seattle DataFusion meetup in early 2025? Maybe Jan/Feb time range?

**Comment by alamb** [comment]
I will be in Seattle and we are organizing a meetup on April 23, 2026. I am confirming the final details but it will likely be in Bellevue 

We are (as always) looking for speakers. Here is a call for speakers (based on the previous one in https://github.com/apache/datafusion/discussions/10348)

We are looking for speakers to present at a Apache DataFusion meetup in Seattle. As in past events, we will have slots for everyone who wants to present. 

If you have a ~15 minute presentation about DataFusion or want to discuss your favorite data processing tools, we want to hear from you!

Here are some topics to get your creative juices flowing:

🔥 Performance testing and optimization magic
🚀 Rolling out a game-changing new feature in the core library
🐛 The thrill (and agony) of squashing a particularly tricky bug
🛠️ Crafting an amazing product using DataFusion

But don't let this list limit you—if it’s related to DataFusion or the broader Apache Arrow ecosystem and it excites you, we’re all ears!

**Reply by CurtHagenlocher** [reply]
(Presumably this should read April 23, **2026**?)

**Reply by alamb** [reply]
Yes, thank you - I have updated it to say so

**Comment by alamb** [comment]
Here are the details on the meetup

Date: Thursday April 23, 2026
Time 17:30 - 20:00
Location: [Databricks Bellevue Office, 500 108th Ave NE, 26th Floor, Bellevue, WA](https://maps.app.goo.gl/6XBznyLHxGTzisjc9) 
Luma Event Page: https://luma.com/hxshbp0m

**Reply by jiayuasu** [reply]
@alamb Hey Andrew, welcome to Seattle! I'd like to present SedonaDB (which also uses DataFusion!) in Seattle DataFusion meetup. Where can I submit my abstract?

**Reply by alamb** [reply]
Thanks @jiayuasu ! That is great. I'll add you to the organizing document -- I sent the invite to the emails listed on https://jiayuasu.github.io/. If you want a different one, please email me the address you would like to use at alamb@influxdata.com

**Reply by jiayuasu** [reply]
thanks @alamb ! I've updated the doc accordingly. Looking forward to it!

**Reply by CurtHagenlocher** [reply]
There's glitch on the Luma signup page where the time at the top shows up for me as 2:30 PM - 5:00 PM. I'm guessing there's some kind of time zone issue involved. The Agenda **does** show the event running from 5:30 PM to 8:00 PM.

**Reply by alamb** [reply]
Thank you @CurtHagenlocher  -- I fixed the times on the invite. I think it is right now:


<img width="998" height="389" alt="Screenshot 2026-02-09 at 10 29 29 AM" src="https://github.com/user-attachments/assets/04454e55-9948-4e1a-aa32-3c00d8c7f782" />

**Reply by jiayuasu** [reply]
@alamb Hey Andrew, could you update the Luma event page to add my name and presentation there? Thank you! 😁

**Reply by tbar4** [reply]
Dang! The one week in April I won't be able to go! Hopefully they become more routine.

**Reply by alamb** [reply]
> @alamb Hey Andrew, could you update the Luma event page to add my name and presentation there? Thank you! 😁

Done @jiayuasu

**Reply by waynexia** [reply]
Hi, thank you for hosting this event! I'd like to share my recent work on implementing a JSON type (repo: https://github.com/waynexia/jsonfusion) with DataFusion if there is a slot left. 

(is this reply a submission or it should be posted somewhere else

**Reply by alamb** [reply]
I'll add you to th elist and speaker chain shortly

**Reply by alamb** [reply]
I added you to the list / email chain

BTW this discussion might be of interest to you as well: https://github.com/apache/datafusion/issues/21301

**Reply by newfront** [reply]
Thanks for having me as well.

**Comment by alamb** [comment]
Here are the videos from the event

* [Scott Haines: Unifying Open Lakehouse Governance via Policy Portability, Seattle/Bellevue April 2024](https://youtu.be/WMhaqR5pgYU)
* [Ruihang Xia: JSONFusion Columnar JSON on Apache DataFusion + Parquet, Seattle/Bellevue April 2024](https://youtu.be/YFxei9kFMDE)
* [Jai Yu: Spatial Analytics on DataFusion, Seattle/Bellevue April 2024](https://youtu.be/fQnhuu4A0uQ)
* [Luke Kim: The Cluster Sidecar Architecture with Apache DataFusion, Seattle/Bellevue April 2024](https://youtu.be/X8YM20sH8vc)
* [Weston Pace: DataFusion without Row Groups, Seattle/Bellevue April 2024](https://youtu.be/fDrmfDuPK3s)
* [Andrew Lamb: Intro to Apache DataFusion, Seattle/Bellevue April 2024](https://youtu.be/wwhKDYIGM4s)

**Comment by alamb** [comment]
Here are some photos of the Event (thanks to Wyatt!)

<img width="1024" height="768" alt="Weston" src="https://github.com/user-attachments/assets/cf37fd6a-ab05-4dfa-99f8-e9dc98cac436" />
<img width="1024" height="768" alt="Meeting up" src="https://github.com/user-attachments/assets/23d492c2-9e3d-4b4a-bec2-89ac62713b1b" />
<img width="1024" height="768" alt="Scott" src="https://github.com/user-attachments/assets/3154f8f0-b469-4520-bc26-73781b9165fe" />
<img width="1024" height="768" alt="Andrew" src="https://github.com/user-attachments/assets/d7a8fecd-f93f-4ef5-a024-bd80ce9739da" />
<img width="1024" height="768" alt="Jia" src="https://github.com/user-attachments/assets/6c743f32-685d-4ac7-95cd-7247fd055c7a" />
<img width="1024" height="768" alt="Ruihang" src="https://github.com/user-attachments/assets/33a0ad59-8af8-4cde-b8f0-3436dd239a30" />

**Comment by alamb** [comment]
More pictures from Cole Bowdin:

<img width="4284" height="5712" alt="IMG_5405" src="https://github.com/user-attachments/assets/1644cf74-6992-46d4-9af6-2a260edc3c64" />
<img width="3024" height="4032" alt="IMG_5406" src="https://github.com/user-attachments/assets/87149b10-76ef-4c70-ab2b-9716ba083c52" />
<img width="4284" height="5712" alt="IMG_5407" src="https://github.com/user-attachments/assets/2678d4dc-f0d3-4cde-8665-4af5b27722b5" />
<img width="4284" height="5712" alt="IMG_5408" src="https://github.com/user-attachments/assets/be9f964e-fab2-4e9a-a93f-7c07a26128e2" />
<img width="5712" height="4284" alt="IMG_5409" src="https://github.com/user-attachments/assets/d8997fca-c321-4675-966c-8096210de6c6" />
<img width="4032" height="3024" alt="IMG_5410" src="https://github.com/user-attachments/assets/9cc27508-93fa-44cf-969a-cb8a3e38ff51" />
<img width="4032" height="3024" alt="IMG_5411" src="https://github.com/user-attachments/assets/1db72742-1e75-49f5-ba87-bf7327d883c6" />
